charts/kuadrant-operator/Chart.yaml

```diff
@@ -30,7 +29,7 @@ dependencies:
     version: 0.0.0
     repository: https://kuadrant.io/helm-charts/
   - name: dns-operator
-    version: 0.0.0
+    version: remove_managed_zone_api
```

Comment:
Was this the result of `make helm-build` with no args?

Reply:
For main branch, it should be `0.0.0` then for the releases it points to the actual released dependency

Reply:
hmm I am not sure actually. But I will change it

Reply:
done 👍